terraform/modules/benchmark/service/run_benchmark.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+## Enable strict error handling
+set -eux
+
+# 1. Initialize s3cmd
+# 2. Run benchmark
+# 3. Upload benchmark results to s3 bucket
+
+## Configure s3cmd
+CFG=~/.s3cfg
+
+if [ ! -f "$CFG" ]; then
+    s3cmd --dump-config \
+        --host="$BENCHMARK_ENDPOINT" \
+        --host-bucket="%(bucket)s.$BENCHMARK_ENDPOINT" \
+        --access_key="$AWS_ACCESS_KEY_ID" \
+        --secret_key="$AWS_SECRET_ACCESS_KEY" \
+        --multipart-chunk-size-mb=4096 > "$CFG"
+    echo "Configuration file created at $CFG"
+else
+    echo "s3cmd Configuration file exist at $CFG"
+fi
+
+## Run actual benchmark
+ruby bench.rb --chain calibnet --tempdir ./tmp --daily
+ruby bench.rb --chain mainnet --tempdir ./tmp --daily
+
+## Upload benchmark result to s3 weekly file
+week_number=$(date +%W) # Week starting on Monday
```

Comment:
Is overwriting benchmarks from January 2023 with results from January 2024 expected and by design?

Reply:
added year to avoid this condition